config/routes.rb

```diff
@@ -4,4 +4,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-end
+
+  
+    resources :users do
+      resources :posts
+    end
```

Comment:
- [ ] Kindly ensure your route config handles only the 4 required URLs, by specifying the routes to show in your resources.